make-zip

```diff
@@ -33,11 +33,12 @@ assert os.path.isdir( BASE_DIR )
 # Temporary directory for constructing everything inside
 tmpd = tempfile.mkdtemp( suffix="-pyenv" )
 
-subprocess.check_call( "cp -r %s/* %s" % (BASE_DIR, tmpd),
-                       shell = True )
+# annoyingly, copytree creates the directory instead of merging in Python 2
```

Comment:
We should make this script Python 3 only at some point.

Reply:
-> #2